README.md

```diff
@@ -41,16 +41,18 @@ How to use?
         Port "dpdk1"
             Interface "dpdk1"
                 type: dpdk
-# clang -O2 -target bpf -c examples/bpf/rate-limiter.c -o /tmp/rate-limiter.o
-# ovs-ofctl load-filter-prog br0 1 /tmp/rate-limiter.o
-# ovs-ofctl add-flow br0 priority=1,in_port=1,filter_prog=1,actions=output:2
-# ovs-ofctl dump-flows br0
+$ clang -O2 -target bpf -c examples/bpf/dropper.c -o /tmp/dropper.o
+$ ovs-ofctl load-filter-prog br0 1 /tmp/dropper.o
+$ ovs-ofctl add-flow br0 priority=1,in_port=1,filter_prog=1,actions=output:2
+$ ovs-ofctl add-flow br0 priority=1,in_port=2,actions=output:1
+$ ovs-ofctl dump-flows br0
 NXST_FLOW reply (xid=0x4):
  cookie=0x0, duration=103.730s, table=0, n_packets=0, n_bytes=0, idle_age=103, priority=1,in_port=1,filter_prog=1 actions=output:2
- cookie=0x0, duration=103.842s, table=0, n_packets=0, n_bytes=0, idle_age=103, priority=0 actions=NORMAL
+ cookie=0x0, duration=103.842s, table=0, n_packets=0, n_bytes=0, idle_age=103, in_port=2,actions=output:1
+# Drop (value=1) packets destined to IP 172.16.0.14.
```

Comment:
Actually, we probably should explain the 1 and 0 too: `Drop (value=1) packets destined to IP 172.16.0.14 through map 0 of filter_prog 1.`? Sorry I didn't see this before.